src/qt/utilitydialog.cpp

```diff
@@ -21,7 +21,9 @@
 #include <clientversion.h>
 #include <init.h>
 #include <interfaces/node.h>
+#include <util.h>
 #include <util/system.h>
+#include <utilstrencodings.h>
```

Comment:
14521:
```suggestion
#include <util/strencodings.h>
```

Reply:
Ha! I was on the wrong branch!! That's why it had a different name 🤯 

Thanks for catching that.